base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -246,10 +310,17 @@ static int send_close_messages(void)
         return 1;
     }
 
+    pkill_list = HeapAlloc(GetProcessHeap(), 0, pid_list_size * sizeof(DWORD));
+    if (!pkill_list)
+        return 1;
+
     for (i = 0; i < task_count; i++)
     {
         WCHAR *p = task_list[i];
         BOOL is_numeric = TRUE;
+        DWORD pkill_size = 0, index = 0;
+
+        memset(pkill_list, 0, pid_list_size * sizeof(DWORD));
```

Comment:
Just a small suggestion... why not specify HEAP_ZERO_MEMORY in HeapAlloc?
then there's no need to memset0

Reply:
There is the for-loop, and if one iteration fills the pkill_list but then the contents need to be discarded for the next loop iteration, then the memset would be still needed.

Reply:
Uh... you are right. Sorry for my mistake